docs/book/psr16.md

```diff
@@ -15,12 +15,15 @@ operations.
 Instantiation is as follows:
 
 ```php
-use Laminas\Cache\StorageFactory;
 use Laminas\Cache\Psr\SimpleCache\SimpleCacheDecorator;
+use Laminas\Cache\Service\StorageAdapterFactoryInterface;
+use Psr\Container\ContainerInterface;
 
-$storage = StorageFactory::factory([
-    'adapter' => 'apc',
-]);
+/** @var ContainerInterface $container */
+$container = null; // can be any configured PSR-11 container
```

Comment:
@boesing 
> Should I refer to the `Standalone` section to provide a way how the `container` could be "retrieved"?

Stand-alone is correct because the component can be used outside of laminas-mvc or Mezzio based application. The basic usage and all other examples should follow this fact.

The integration in a laminas-mvc, Mezzio or other application must be described under the section "Application Integration". (laminas-mvc controller plugins and similar packages are clear exceptions)

Reply:
@froschdesign 

Hm, I still dont really know how to provide proper examples for the "config-driven" instantiations.

With v3.0+, manual instantiation will still be possible but the config-driven approach wont work without having a PSR-11 container with the new `StorageAdapterFactoryInterface` configured, e.g.

How could I address that properly? I also don't get how an "Application Integration" section might look like.